src/main/java/strategy/operations/impl/BalanceOperation.java

```diff
@@ -0,0 +1,18 @@
+package strategy.operations.impl;
+
+import dao.StorageDao;
+import model.Fruits;
+import strategy.operations.OperationHandler;
+
+public class BalanceOperation implements OperationHandler {
```

Comment:
```suggestion
public class BalanceOperationHandlerImpl implements OperationHandler {
```
change each operationhandlerimpl accordingly